networkit/test/test_graph.py

```diff
@@ -90,6 +90,43 @@ def testRemoveMultiEdges(self):
 					self.assertFalse(G.hasEdge(u, v))
 				self.assertEqual(G.numberOfEdges(), 0)
 
+	def testGraphFromGraphEventStream(self):
+		up1 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.NODE_ADDITION, 0, 0, 0)	
+		up2 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.NODE_ADDITION, 1, 1, 1)	
+		up3 = nk.dynamics.GraphEvent(nk.dynamics.GraphEventType.NODE_ADDITION, 2, 2, 2)	
+		self.assertIsInstance(up1.__repr__(),str)
+		self.assertFalse(up1.__eq__(up2))
+		self.assertTrue(up1.__ne__(up2))
+		self.assertTrue(up1.__lt__(up2))
+		self.assertFalse(up1.__gt__(up2))
+		self.assertTrue(up1.__le__(up2))
+		self.assertFalse(up1.__ge__(up2))		
```

Comment:
Using `__eq__`, `__ne__`, etc. should not be necessary, they enable using `==`, `<=`, etc. Please use `assertEqual`, `assertLessEqual`, etc.

Reply:
Also, these tests are not related with the GraphEventStream. Please move them to another test.